gui/include/videodecoder.h

```diff
@@ -22,16 +22,32 @@
 
 #include "exception.h"
 
+#include <QMap>
 #include <QMutex>
 #include <QObject>
 
 extern "C"
 {
 #include <libavcodec/avcodec.h>
+#include <libswscale/swscale.h>
```

Comment:
```suggestion
```

Reply:
Yea just noticed that, sorry. My local build of ffmpeg has it all so it was not failing. Fixing now.